sails/api/hooks/custom/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * custom hook
```

Comment:
What does this "custom hook" do? Better description and rename the hook folder to match the use case.